static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js

```diff
@@ -103,13 +105,32 @@ define([
         }
     }
 
+    function initMostPopularCard(el) {
+        var height;
+        var $adSlot = $(el);
+        var $mostPopTabs;
+
+        if ($adSlot.hasClass('ad-slot--mostpop')) {
+            fastdom.read(function () {
+                height = $adSlot.dim().height;
+                $mostPopTabs = $('.js-most-popular-footer .tabs__pane');
+            });
+
+            fastdom.write(function () {
+                $mostPopTabs.css('height', height);
+            });
+        }
+
+        createToggle(el);
+    }
+
     CommercialComponent.prototype.postLoadEvents = {
         bestbuy: function (el) {
             new Tabs().init(el);
         },
         capi: createToggle,
         capiSingle: createToggle,
-        paidforCard: createToggle
+        paidforCard: initMostPopularCard
```

Comment:
This tight coupling between `createToggle` and `initMostPopularCard` should stay out of each function's logic. We could compose them, with the only contract that each function must return the element:

`paidforCard: compose(createToggle, initMostPopularCard)`